src/components/pages/AdminAddCourse/AdminAddCoursesForm.js

```diff
@@ -1,8 +1,10 @@
 import React, { useState } from 'react';
+import axiosWithAuth from '../../../utils/axiosWithAuth';
```

Comment:
Let's comment these out with a commented `TODO` to implement it correctly once `axiosWithAuth` works with Auth0 - we've got a lot of Linter Warnings for unused definitions for this function currently.